apps/passport-client/src/appHooks.ts

```diff
@@ -362,3 +362,35 @@ export function useZapp(): Zapp | undefined {
 export function useZappOrigin(): string | undefined {
   return useSelector<string | undefined>((s) => s.zappOrigin, []);
 }
+
+// Fixes an issue on iOS where components break when switching from landscape to portrait mode.
+// Relevant issue: https://linear.app/0xparc-pcd/issue/0XP-1334/when-changing-to-landscape-and-than-back-to-portrait-the-home-page
+// This hook detects orientation changes and forces a reflow by briefly hiding and redisplaying the document body,
+// ensuring components are re-rendered correctly.
+// https://linear.app/0xparc-pcd/issue/0XP-1334/when-changing-to-landscape-and-than-back-to-portrait-the-home-page
+export const useIOSOrientationFix = (): void => {
```

Comment:
is this hook invoked everywhere it needs to be? where does the issue that it fixes present specifically? should it be used on all pages or just some of them? could this be added to the `index.tsx` file of `passport-client`?

Reply:
empirically it seems to not be an issue from all the screens i tested, but noted it here

https://linear.app/0xparc-pcd/issue/0XP-1475/should-useiosorientationfix-be-in-indextsx